src/List/ListItem.js

```diff
@@ -70,13 +70,23 @@ export default class ListItem extends Component {
       PropTypes.string,
       PropTypes.func,
     ]),
+    /**
+     * If true, compact vertical padding designed for keyboard and mouse input will be used.
+     */
     dense: PropTypes.bool,
     /**
      * @ignore
      */
     disabled: PropTypes.bool,
+    /**
+     * If true, a 1px light border is added to the bottom of the list item.
+     */
     divider: PropTypes.bool,
-    gutters: PropTypes.bool,
+    /**
+     * If `false`, the left and right padding is removed.
+     */
+    gutters: PropTypes.bool, // TODO: Technically not 'gutters'.
```

Comment:
I'm also having a concern with the default property being `true`. Hence I would rather use the negative: `disableGutters` as I have been writing down on the API consistency section.